.github/workflows/watch-conda-forge.yml

```diff
@@ -3,6 +3,7 @@ name: Check for new versions of Dask on Conda Forge
 on:
   schedule:
     - cron: "0 * * * *"
+  pull_request:
```

Comment:
Using this to make sure the outputs are expected; will remove once things are working properly